0000-template.md

```diff
@@ -1,23 +1,30 @@
 ---
-Stage: Accepted
-Start Date: 
-Release Date: Unreleased
-Release Versions:
-  ember-source: vX.Y.Z
-  ember-data: vX.Y.Z
-Relevant Team(s): 
-RFC PR: 
+stage: accepted
+start-date: 
+release-date: 
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: # update this to the PR that you propose your RFC in
+project-link:
 ---
 
 <!--- 
 Directions for above: 
 
-Stage: Leave as is
-Start Date: Fill in with today's date, YYYY-MM-DD
-Release Date: Leave as is
-Release Versions: Leave as is
-Relevant Team(s): Fill this in with the [team(s)](README.md#relevant-teams) to which this RFC applies
-RFC PR: Fill this in with the URL for the Proposal RFC PR
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+proposal-pr: Fill this in with the URL for the Proposal RFC PR
+tracking-link: Leave as is
```

Comment:
this has been removed or renamed

Reply:
sorry I missed the comment block directly inside the content of the markdown 🙈 I'll fix this now